templates/shortcode-receipt.php

```diff
@@ -77,45 +71,67 @@
 		</tr>
 		<?php endif; ?>
 
-		<?php if ( ! empty( $order->get_fees() ) ) : ?>
-		<tr>
-			<td><strong><?php esc_html_e( 'Fees', 'easy-digital-downloads' ); ?>:</strong></td>
-			<td>
-				<ul class="edd_receipt_fees">
-				<?php foreach ( $order->get_fees() as $fee ) : ?>
-					<li>
-						<?php if ( 'order_item' === $fee->object_type || ! empty( $fee->description ) ) : ?>
-							<span class="edd_fee_label">
-								<?php
-								if ( 'order_item' === $fee->object_type ) {
-									$order_item = edd_get_order_item( $fee->object_id );
-									if ( $order_item ) {
-										echo esc_html( $order_item->product_name ) . ':';
-									}
-								}
-								?>
-							<?php if ( ! empty( $fee->description ) ) : ?>
-								<?php echo esc_html( $fee->description ); ?>
-							<?php endif; ?>
-							</span>
-							<span class="edd_fee_sep">&nbsp;&ndash;&nbsp;</span>
-						<?php endif; ?>
-						<span class="edd_fee_amount"><?php echo esc_html( edd_currency_filter( edd_format_amount( $fee->subtotal ) ) ); ?></span>
-					</li>
-				<?php endforeach; ?>
-				</ul>
-			</td>
-		</tr>
+		<?php if ( filter_var( $edd_receipt_args['price'], FILTER_VALIDATE_BOOLEAN ) && $order->subtotal > 0 ) : ?>
+			<tr>
+				<td><strong><?php esc_html_e( 'Subtotal', 'easy-digital-downloads' ); ?>:</strong></td>
+				<td>
+					<?php echo esc_html( edd_payment_subtotal( $order->id ) ); ?>
+				</td>
+			</tr>
 		<?php endif; ?>
 
-		<?php if ( filter_var( $edd_receipt_args['discount'], FILTER_VALIDATE_BOOLEAN ) && ! empty( $user['discount'] ) && 'none' !== $user['discount'] ) : ?>
+		<?php
+		if ( filter_var( $edd_receipt_args['discount'], FILTER_VALIDATE_BOOLEAN ) ) :
+			$discounts = $order->get_discounts();
+			if ( $discounts ) :
+				$label = _n( 'Discount', 'Discounts', count( $discounts ), 'easy-digital-downloads' );
+				?>
+				<tr>
+					<td><strong><?php echo esc_html( $label ); ?>:</strong></td>
+					<?php
+					foreach ( $discounts as $discount ) {
+						$name   = $discount->description;
+						$amount = edd_currency_filter( edd_format_amount( edd_negate_amount( $discount->total ) ) );
+						if ( 'percent' === edd_get_discount_type( $discount->type_id ) ) {
+							$rate  = edd_format_discount_rate( 'percent', edd_get_discount_amount( $discount->type_id ) );
+							$name .= "&nbsp;({$rate})";
+						}
+						?>
+						<td><?php echo esc_html( $name . ': ' . $amount ); ?></td>
```

Comment:
I'm not sure how to remedy this, but right now with multiple discounts the table cells are next to each other rather than stacked in 3.0.  
![image](https://user-images.githubusercontent.com/18077352/116151217-3091ca00-a699-11eb-8839-3e9a0ec8d5e4.png)

Reply:
That's because I apparently had the `<td>` inside of the discounts `foreach`. I went to fix it and ended up changing discounts and fees completely to match what Invoices does--a separate row for each discount/fee, with the label in the left column and the total in the right column. It feels easier to read to me with this instead of the `<ul>`.

Reply:
Closer, but I'm still seeing the discounts side by side rather than stacked.  It shouldn't matter, but the 5% discount is from DCG.
![image](https://user-images.githubusercontent.com/18077352/116441123-af753700-a840-11eb-9995-48069be87bbf.png)